<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Stephens-MBP.attlocal.net" ignored="0" name="Default test" tests="11" failures="0" timestamp="13 Nov 2020 19:10:21 GMT" time="4.396" errors="0">
  <testcase name="@BeforeClass mockSetup" time="0.001" classname="com.pizzeria.training.services.PizzaServiceTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.pizzeria.training.services.PizzaServiceTest.mockSetup(PizzaServiceTest.java:41)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass mockSetup -->
  <testcase name="getAll" time="0.146" classname="com.pizzeria.training.controllers.PizzaControllerIntegrationTest"/>
  <testcase name="getTest" time="0.005" classname="com.pizzeria.training.controllers.PizzaControllerIntegrationTest"/>
  <testcase name="findAll" time="0.157" classname="com.pizzeria.training.repositories.PizzaRepositoryIntegrationTest"/>
  <testcase name="save" time="0.012" classname="com.pizzeria.training.repositories.PizzaRepositoryIntegrationTest"/>
  <testcase name="findAll" time="0.0" classname="com.pizzeria.training.services.PizzaServiceTest">
    <skipped/>
  </testcase> <!-- findAll -->
  <testcase name="save" time="0.0" classname="com.pizzeria.training.services.PizzaServiceTest">
    <skipped/>
  </testcase> <!-- save -->
  <testcase name="allArgsConstructor" time="0.0" classname="com.pizzeria.training.models.PizzaModelTestNGTest"/>
  <testcase name="equalsHashcode" time="0.002" classname="com.pizzeria.training.models.PizzaModelTestNGTest"/>
  <testcase name="noArgsConstructor" time="0.001" classname="com.pizzeria.training.models.PizzaModelTestNGTest"/>
  <testcase name="setters" time="0.001" classname="com.pizzeria.training.models.PizzaModelTestNGTest"/>
  <testcase name="toStringTest" time="0.001" classname="com.pizzeria.training.models.PizzaModelTestNGTest"/>
</testsuite> <!-- Default test -->
